python/dgl/heterograph.py

```diff
@@ -2634,67 +3774,180 @@ def edge_attr_schemes(self, etype=None):
         return self._edge_frames[self.get_etype_id(etype)].schemes
 
     def set_n_initializer(self, initializer, field=None, ntype=None):
```

Comment:
We are not deprecating the `set_*_initializer` functions?

Reply:
No.

Reply:
I guess our docstrings don't have to talk about initializers...